templates/repo/issue/milestones.tmpl

```diff
@@ -64,6 +64,7 @@
 						<span class="issue-stats">
 							<i class="octicon octicon-issue-opened"></i> {{$.i18n.Tr "repo.issues.open_tab" .NumOpenIssues}}
 							<i class="octicon octicon-issue-closed"></i> {{$.i18n.Tr "repo.issues.close_tab" .NumClosedIssues}}
+							{{if and .TotalTimes .MustGetRepo.IsTimetrackerEnabled}}<i class="octicon octicon-clock"></i> {{.TotalTimes}}{{end}}
```

Comment:
Same problem here as it was with issue list (each milestone queried even if time tracking disabled) and also each milestone queried separately